src/app/cli/src/commands/system_diagnose_command.rs

```diff
@@ -316,21 +374,26 @@ impl DiagnosticCheck for CheckWorkspaceConsistent {
 
 ///////////////////////////////////////////////////////////////////////////////
 
-fn handle_output_result(result: Output) -> Result<(), DiagnosticCheckError> {
+fn handle_output_result(
+    result: Output,
+    stderr_file_path: Vec<PathBuf>,
```

Comment:
It seems it's always 1 stderr path, where do u need a vector?

Reply:
It was done like in another place, where we were passing `stdout`result. I would like to keep it, because in future we may need to catch info from processes printed in `stdout`